Tests/webpack.config.js

```diff
@@ -0,0 +1,72 @@
+const path = require('path');
+const webpack = require('webpack');
+
+module.exports = {
+  target: 'web',
+  mode: 'development', // or 'production'
+  devtool: false,
+  entry: {
+    tests: './UnitTests/Scripts/tests.ts',
+  },
+  output: {
+    filename: '[name].js',
+    path: path.resolve(__dirname, 'UnitTests/dist'),
+  },
+  plugins: [
+    new webpack.ProvidePlugin({
+    process: 'process/browser',
+    Buffer: ['buffer', 'Buffer'],
+  }),
+    new webpack.optimize.LimitChunkCountPlugin({
+      maxChunks: 1, // ensures all code is in a single chunk
+    }),
+  ],
+  resolve: {
+    extensions: ['.ts', '.js'],
+    fallback: {
+        'fs': false,
+        'path': false,
+        'util': false,
+        'process': false,
+        'buffer': false,
+        stream: require.resolve('stream-browserify'),
+        'events': false,
+        'assert': false,
+        'crypto': false,
+        'url': false,
+        'querystring': false,
+        'http': false,
+        'https': false,
+        'zlib': false,
+        'tty': false,
+        'os': false,
+        'child_process': false,
+        'net': false,
+        'dgram': false,
+        'dns': false,
+        'readline': false,
+        'repl': false,
+        'cluster': false,
+        'vm': false,
+        'domain': false,
+        'constants': false,
+        'module': false,
+        'timers': false
+    }
+  },
+  module: {
+    rules: [
+        {
+          test: /\.(js|ts)$/,
+          exclude: (modulePath) => {
+            return (
+              /node_modules/.test(modulePath) &&
+              !/node_modules[\\/](?:@babylonjs|mocha|chai)/.test(modulePath)
+            );
+          },
+          use: 'babel-loader',
```

Comment:
I think you can put Babel configuration inline here, rather than separate babel specific files. Would this make more sense?

If you prefer to keep it in a separate file, maybe `babel.config.js` (instead of json or rc) since it is more flexible?